examples/knative/powershell/kn-ps-harbor-slack/function.yaml

```diff
@@ -30,8 +30,7 @@ spec:
   broker: default
   filter:
     attributes:
-      type: com.vmware.harbor.push_artifact.v0
-      subject: admin
+      type: harbor.artifact.pushed
```

Comment:
We should follow our naming convention for events, so using the previous one it should now be `com.vmware.harbor.pushed.v0`

Reply:
This is an original event coming from Harbor @lamw, therefore we can't change it as we like.

Reference: https://goharbor.io/docs/2.8.0/working-with-projects/project-configuration/configure-webhooks/ 

![image](https://github.com/vmware-samples/vcenter-event-broker-appliance/assets/31652019/a8d5950b-0f06-4ef6-877c-59d0c9abd0da)

Reply:
I guess it would have been nice if they could have conformed to standard as mentioned above

Reply:
Agree!